applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -752,7 +759,6 @@ int slm_at_host_init(void)
 			LOG_ERR("slm_setting_uart_save: %d", err);
 			return err;
 		}
-		uart_configured = true;
 	} else {
 		/* else re-config UART based on setting page */
```

Comment:
Have you confirmed #XSLEEP=0 wakeup of two setup?
- UART0 without HWFC
- UART2 with HWFC
I don't see how the wakeup issue is fixed by change the timing of "uart_configured = true"

Reply:
As we discussed , this is another bug fix.
The actually fix for DK is not to disconnect HWFC pin.